### ISSUE TITLE
[FR] Consistent image alt text render

### ISSUE THREAD

**Issue by r-leyshon** [rstudio/rmarkdown#2374]
Accessibility audit pointed to unreliable rendering of image alt text when using markdown syntax in Rmd documents knitted to html. At times, the markdown syntax renders correctly - image with non-visible alt text available to screen readers. On occasion, the image renders with visible alt text. Screen readers would then read the alt text twice as the visible and non-visible alt text tags. This is an accessibility error and results in navigation issues with screen reader users tabbing through the page tags. 

Also, no alt text is included with charts as standard.

The [approach taken in `accessrmd::access_img()` ](https://github.com/datasciencecampus/accessrmd/blob/main/R/access_img.R) is only compatible with `ggplot2` charts and images on disk. That is to say, base plot compatibility has not yet been achieved. But the idea is to write ggplot charts to tempfile, then read as if it were any other disk image, embedding the filepath within a html img tag. 

The html img tag approach passed our accessibility audit, we just found that most of our colleagues favoured the less reliable markdown syntax and needed a faster way for them to write code that would pass our service standard.

A good website for checking these issues is [WAVE webaim](https://wave.webaim.org/). You can also use the chrome WAVE plugin. Opening your rendered html docs within the WAVE interface is a good first start for troubleshooting many of the WCAG2.1 'gotchas'.
